src/login/logind.c

```diff
@@ -1036,7 +1036,7 @@ static int manager_dispatch_idle_action(sd_event_source *s, uint64_t t, void *us
                         else
                                 log_info("System idle. Will %s now.", handle_action_verb_to_string(m->idle_action));
 
-                        manager_handle_action(m, 0, m->idle_action, false, is_edge);
+                        manager_handle_action(m, 0, m->idle_action, false, is_edge, NULL);
```

Comment:
as per our current coding style: if you call a function with a null/zero param were it isn#t clear what this is, please prefix with the parameter name in a comment. i.e. `/* action_seat= */` here.

(and if you want to be nice, you do the same for the 2nd and 4th argument too, since they are also completely opaque.)

(our general rule is that those who touch a line of code need to modernize it to current coding style)

(current coding style is documented here btw: https://systemd.io/CODING_STYLE)